include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -218,15 +225,18 @@ struct Allocator : Base<OrtAllocator> {
 };
 
 struct AllocatorInfo : Base<OrtAllocatorInfo> {
-  static AllocatorInfo Create_Cpu(OrtAllocatorType type, OrtMemType mem_type1);
+  static AllocatorInfo CreateCpu(OrtAllocatorType type, OrtMemType mem_type1);
 
   AllocatorInfo(nullptr_t) {}
```

Comment:
There's a lot of single argument constructors like this one which could be marked explicit

Reply:
Good idea, fixing for the next pr update